index.d.ts

```diff
@@ -523,6 +524,9 @@ declare namespace CbServer {
     newCollection(name: string): Promise<{ id: string; name: string }>;
     Database(options?: { externalDBName: string }): DatabaseAsync;
   }
+  interface CacheAsync<T = unknown> {
```

Comment:
im not too familiar with async cache, but is there a reason this is defaulting to unknown instead of object but everywhere else it's extending an object? Also, does the getAll method just return that one interface or an array of interfaces?